### DISCUSSION TITLE
Is it possible to get a wxpython 4.1.1 formula?

### DISCUSSION THREAD

**Discussion by briantoby** [Homebrew/discussions#4053] [Tap maintenance and Homebrew development] [answered]
At present there is a bug in wx 4.2.0 (https://github.com/wxWidgets/Phoenix/issues/2319) but it does not seem very easy to regress to a 4.1.1 build. It would be great to be able to request wxpython@4.1.

**Comment by SMillerDev** [comment, marked as answer]
Unless upstream supports the release Homebrew will not be making a versioned formula out of it.

**Comment by briantoby** [comment]
As it turns out, the bugs in 4.1.1 w/Python 3.10+ are worse than what I saw with 4.2.0, so there is no reason to offer it as an option.
